content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -91,6 +91,10 @@ Intelligent Test Runner is only supported in the following versions and testing
 * `jest>=24.8.0`
   * Only `jest-circus/runner` is supported as `testRunner`.
   * Only `jsdom` and `node` are supported as test environments.
+* `mocha>=5.2.0`
```

Comment:
Is mocha supported since dd-trace 3.4.0? If not, you'll need to change that.

Reply:
yeah good catch, thanks